scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/ByteCompressionUtils.java

```diff
@@ -0,0 +1,61 @@
+package org.ow2.proactive.scheduler.util;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.zip.DataFormatException;
+import java.util.zip.Deflater;
+import java.util.zip.Inflater;
+
+import org.apache.log4j.Logger;
+
+/**
+ * 
+ * @author ActiveEon team
+ *
+ */
+public class ByteCompressionUtils {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1118)